velox/docs/functions/spark/conversion.rst

```diff
@@ -258,4 +258,23 @@ Valid example
   SELECT cast(1727181032 as timestamp); -- 2024-09-24 12:30:32
   SELECT cast(9223372036855 as timestamp); -- 294247-01-10 04:00:54.775807
   SELECT cast(-9223372036855 as timestamp); -- 290308-12-21 19:59:05.224192
+
+From floating-point types
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Casting from floating-point input to timestamp type is allowed.
+The input value is treated as the number of seconds since the epoch (1970-01-01 00:00:00 UTC) and converted to microseconds.
```

Comment:
@boneanxs Would you clarify whether input is rounded down / up or half-up/down?

Reply:
update the doc here which is `truncate the fractional part`

```c++
static_cast<int64_t>(seconds * Timestamp::kMicrosecondsInSecond)
```